lms/envs/common.py

```diff
@@ -3432,6 +3432,12 @@ def _make_locale_paths(settings):  # pylint: disable=missing-function-docstring
 COURSE_ABOUT_VISIBILITY_PERMISSION = 'see_exists'
 
 DEFAULT_COURSE_VISIBILITY_IN_CATALOG = "both"
+
+# .. toggle_name: DEFAULT_MOBILE_AVAILABLE
+# .. toggle_implementation: DjangoSetting
+# .. toggle_default: False
+# .. toggle_description: This specifies if the courses are
```

Comment:
In order of importance:
**Required:** You are missing some required annotations.  See the [annotation definitions](https://github.com/edx/code-annotations/blob/master/code_annotations/contrib/config/feature_toggle_annotations.yaml) for what is not marked optional. See the [how-to for documenting toggles](https://edx.readthedocs.io/projects/edx-toggles/en/latest/how_to/documenting_new_feature_toggles.html) for more help on the additional annotations.
**Very nice to have:** Is there anything we can say or reference regarding how to make individual courses available when this is False? 
**Nit:** Lines could be up to 120 characters, so you don't need to wrap as much.

Reply:
1. added required annotations.
2. added a detailed description to make individual course available for mobile when this is set to False.
3. Now, wrapping lines at 120 chars.